crates/wkg/src/main.rs

```diff
@@ -200,7 +200,6 @@ async fn main() -> anyhow::Result<()> {
         .init();
 
     let cli = Cli::parse();
-    tracing::debug!(?cli);
```

Comment:
I removed this because we're likely to have credentials or other sensitive things in this CLI so just removing it for now. We can add back in some more specific debug logging as this evolves